### ISSUE TITLE
Investigate "0 items selected" message in VoiceOver for MdSelect

### ISSUE THREAD

**Issue by kara** [angular/material2#1670]
Currently in VoiceOver, the select will always read "0 items selected", even if aria-selected, aria-owns, aria-posinset, and aria-setsize are set. It appears to be this way in Material 1 as well.  Needs investigation / fixing.

**Comment by HumanMedia** [comment]
+1 
We really need this fixed as well. Trying to do a site for a major organization for disabled. This is a big problem for us.

**Comment by naomiblack** [comment]
We think this is also related to an issue we're seeing in aXe testing for md-select with child options.

**Comment by naomiblack** [comment]
possibly connected to https://github.com/angular/material2/issues/6382

**Comment by DevVersion** [comment]
This issue seems to be more related to the general accessibility approach of the select. Since the select options are displayed dynamically outside of the `<mat-select>` element, the options won't be found by screenreaders if the panel is closed.

This means that the `aria-selected` option can't be found, and there will be always `0 items selected`.

I think that it would be better to keep the list options permanently in the DOM, and also inside of the `<mat-select>` element.

**Comment by jelbourn** [comment]
Tracking via #11083
